.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: scala
```

Comment:
If this is going to Rally I think we're going to want to use CJE instead of travis or anything else.

Reply:
We are currently using travis for all of our open-source Scala projects. We can discuss changing this in a separate thread.

Reply:
TIL, no need to discuss if it's the way we do things for our OSS projects